docs/app-areas.md

```diff
@@ -1,37 +1,28 @@
-An AEA is an intelligent agent whose goal is generating economic value for its owner. It can represent machines, humans (individuals or organisations), or data. 
+The <a href="../agent-oriented-development">agent-oriented development</a> guide presented some of the fundamental attributes of the environments which AEA-based solutions are suited for.
 
-## General application areas
+Under the light of that discussion, in this page we identify a number of application areas for AEA-based solutions. This list is by no means comprehensive. In fact, we are most excited about applications we have not thought of before.
 
-As described in the guide on <a href="../agent-oriented-development">agent-oriented development</a>, AEAs are designed to operate in an environment with:
-
-* multiple stakeholders, which
-* are represented by AEAs, that
-* interact autonomously and
-* communicate _via_ a peer-to-peer network.
-
-There are at least five general application areas for AEAs:
-
-* **Inhabitants**: agents paired with real world hardware devices such as drones, laptops, heat sensors, etc. An example is the thermometer agent that can be found <a href="../thermometer-skills">here</a>.
-* **Interfaces**: facilitation agents which provide the necessary API interfaces for interaction between old (Web 2.0) and new (Web 3.0) economic models. An example is the http skill in <a href="../http-connection-and-skill">this</a> agent.
-* **Pure software**: software agents living in the digital space that interact with inhabitant and interface agents and others.
-* **Digital data sales agents**: pure software agents that attach to data sources and sell it via the open economic framework. An example can be found <a href="../ml-skills">here</a>.
-* **Representative**: an agent which represents an individual's activities on the Fetch.ai network. An example can be found <a href="../tac-skills">here</a>.
+* **Inhabitants**: agents representing objects in the IoT (Internet of Things) space. For examples, AEAs paired with real world hardware devices such as drones, laptops, heat sensors, etc. An example is a <a href="../thermometer-skills"> thermometer agent </a>.
+* **Interfaces**: facilitation agents which provide the necessary API interfaces for interaction between existing (Web 2.0) and new (Web 3.0) economic models. An example is an AEA with <a href="../http-connection-and-skill"> HTTP connection and skill </a> who has the capability to communicate using HTTP.
+* **Pure software**: software agents _living_ in the digital space that interact with interface agents and others.
+* **Digital data sales agents**: software agents that attach to data sources and sell it via the open economic framework. An example can be found <a href="../ml-skills">here</a>.
+* **Representative**: an agent who represents an individual's activities on the Fetch.ai network. An example can be found <a href="../tac-skills">here</a>.
```

Comment:
An agent is not a person. Please keep `which` rather than `who` :)

Reply:
Done